### PR TITLE
ADD content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_u…

### PR DESCRIPTION
**Pull request by YouthLab**

### CODE REVIEW

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_plan.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+<!-- Check which versions are available to upgrade to and validate whether your current cluster is upgradeable. To skip the internet check, pass in the optional [version] parameter. -->
+检查当前可供升级的版本有哪些以及验证你当前的集群是否可以升级。传入可选的 [version] 参数可以跳过联网检查。
+
+```
+kubeadm upgrade plan [version] [flags]
+```
+
+<!-- ### Options -->
+### 可选项
```
Comment (pigletfly) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_plan.md]: 选项好还是可选项好？

Comment (tengqm) [reply content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_plan.md]: 嗯，选项 +1

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_plan.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+    <col span="1" />
+  </colgroup>
+  <tbody>
+
+    <tr>
+      <td colspan="2">--allow-experimental-upgrades</td>
+    </tr>
+    <tr>
+      <!-- <td></td><td style="line-height: 130%; word-wrap: break-word;">Show unstable versions of Kubernetes as an upgrade alternative and allow upgrading to an alpha/beta/release candidate versions of Kubernetes.</td> -->
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">显示 Kubernetes 的不稳定版本作为升级的替代方案并且允许升级到 alpha/beta/release 状态的 Kubernetes 候选版本。</td>
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_plan.md]: release candidate 状态，不是 release 状态

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_plan.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+      <!-- <td></td><td style="line-height: 130%; word-wrap: break-word;">Show unstable versions of Kubernetes as an upgrade alternative and allow upgrading to an alpha/beta/release candidate versions of Kubernetes.</td> -->
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">显示 Kubernetes 的不稳定版本作为升级的替代方案并且允许升级到 alpha/beta/release 状态的 Kubernetes 候选版本。</td>      
+    </tr>
+
+    <tr>
+      <td colspan="2">--allow-release-candidate-upgrades</td>
+    </tr>
+    <tr>
+      <!-- <td></td><td style="line-height: 130%; word-wrap: break-word;">Show release candidate versions of Kubernetes as an upgrade alternative and allow upgrading to a release candidate versions of Kubernetes.</td> -->
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">显示 Kubernetes 的 release 候选版本作为升级的替代方案并且允许升级到 release 状态的 Kubernetes 候选版本。</td>
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_plan.md]: here too

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_plan.md (hunk 1 above)
Comment (chenrui333) [reply content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_plan.md]: 我翻译的是 命令行选项 😆

**Comment by chenrui333** [comment]
/hold
due to release-1.13 frozen, https://kubernetes.slack.com/archives/C1J0BPD2M/p1543629408003400

**Comment by chenrui333** [comment]
@YouthLab 
now the release-1.13 is done, we can go back to this PR again

**Comment by chenrui333** [comment]
oh, dev-1.13 is gone probably!!

**Comment by chenrui333** [comment]
it is probably good to directly PR to master since dev-1.13 is done.

/cc @zacharysarah

**Comment by markthink** [comment]
@YouthLab 建议重新提交此PR，冲突太多了

**Comment by markthink** [comment]
@YouthLab 这个PR 冲突太严重了，重新提一个吧

**Pull request closed without merging**